families/supplychain/protos/record.proto

```diff
@@ -16,7 +16,7 @@
 syntax = "proto3";
 
 option java_multiple_files = true;
-option java_package = "sawtooth.config.protobuf";
+option java_package = "sawtooth.supplychain.protobuf";
```

Comment:
This was a copy-paste error - it should `supplychain` like the other protos in this family